pkg/controller/accountclaim/organizational_units.go

```diff
@@ -0,0 +1,209 @@
+package accountclaim
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	awsclient "github.com/openshift/aws-account-operator/pkg/awsclient"
+	controllerutils "github.com/openshift/aws-account-operator/pkg/controller/utils"
+)
+
+// MoveAccountToOU takes care of all the logic surrounding moving an account into an OU
+func MoveAccountToOU(r *ReconcileAccountClaim, reqLogger logr.Logger, accountClaim *awsv1alpha1.AccountClaim, account *awsv1alpha1.Account) error {
+	// aws client
+	awsClient, err := awsclient.GetAWSClient(r.client, awsclient.NewAwsClientInput{
+		SecretName: controllerutils.AwsSecretName,
+		NameSpace:  awsv1alpha1.AccountCrNamespace,
+		AwsRegion:  "us-east-1",
+	})
+
+	// Search for ConfigMap that holds OU mapping
+	instance := &corev1.ConfigMap{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: OUConfigMapName}, instance)
+	if err != nil {
+		// If we failed to retrieve the ConfigMap, simply leave the account in Root
+		unexpectedErrorMsg := fmt.Sprintf("OU: Failed to find OU mapping ConfigMap, leaving account in root")
+		reqLogger.Info(unexpectedErrorMsg)
+		accountClaim.Spec.AccountOU = "ROOT"
+		return r.specUpdate(reqLogger, accountClaim)
+	}
+
+	// Get OU ID for root and base
+	friendlyOUName := account.Spec.LegalEntity.ID
+	baseID, rootID, err := checkOUMapping(instance)
+	if err != nil {
+		invalidOUErrorMsg := fmt.Sprintf("Invalid OU ConfigMap, missing root and/or base fields: %s", instance.Data)
+		reqLogger.Error(err, invalidOUErrorMsg)
+		return err
+	}
+
+	// Log ConfigMap
+	configmapMsg := fmt.Sprintf("OU: OU mapping ConfigMap was found in namespace: %s, root: %s, base: %s", OUConfigMapName, rootID, baseID)
```

Comment:
I'm not sure how helpful this log will be since it will be nearly the same info for every accountcaim.

Reply:
you're right, removing this log